public/app/features/provisioning/api/endpoints.gen.ts

```diff
@@ -1083,29 +1074,3 @@ export type ResourceStats = {
   kind?: string;
   metadata?: any;
 };
-export const {
-  useListJobQuery,
-  useGetJobQuery,
-  useListRepositoryQuery,
-  useCreateRepositoryMutation,
-  useGetRepositoryQuery,
-  useReplaceRepositoryMutation,
-  useDeleteRepositoryMutation,
-  useCreateRepositoryExportMutation,
-  useGetRepositoryFilesQuery,
-  useGetRepositoryFilesWithPathQuery,
-  useReplaceRepositoryFilesWithPathMutation,
-  useCreateRepositoryFilesWithPathMutation,
-  useDeleteRepositoryFilesWithPathMutation,
-  useGetRepositoryHistoryQuery,
-  useGetRepositoryHistoryWithPathQuery,
-  useGetRepositoryResourcesQuery,
-  useGetRepositoryStatusQuery,
-  useReplaceRepositoryStatusMutation,
-  useCreateRepositorySyncMutation,
-  useCreateRepositoryTestMutation,
-  useGetRepositoryWebhookQuery,
-  useCreateRepositoryWebhookMutation,
-  useGetFrontendSettingsQuery,
-  useGetResourceStatsQuery,
-} = injectedRtkApi;
```

Comment:
??? is this OK?  things all seem to still work

Reply:
No, this needs to be there, not sure how it got deleted.